### PR TITLE
Create footer styles

### PR DESCRIPTION
**Pull request by alexcracea**

### CODE REVIEW

File: scss/_footer.scss [hunk 1]
```diff
@@ -1,3 +1,6 @@
 .footer {
-
```
Comment (Daniel-Moraru) [inline scss/_footer.scss:2]: i feel very disappoint , you just created an empty class

**Review by PumbaAzaza — COMMENTED** [review]
Tare nicego.

**Review by Daniel-Moraru — CHANGES_REQUESTED** [review]

**Comment by allexei72** [comment]
good boy

**Comment by todirasandrian** [comment]
its still very nice i liked

File: css/styles.css [hunk 2]
```diff
@@ -49,28 +45,8 @@ table {
...
-body {
-  background: green;
-  display: flex;
-  -webkit-appearance: none;
-  -moz-appearance: none;
-  appearance: none;
-  color: #333; }
-
-.box {
-  border: 1px solid #000; }
```
Comment (KateCaz) [inline css/styles.css:61]: Change the border color

**Review by KateCaz — APPROVED** [review]

**Pull request merged**